### PR TITLE
SPCTR-3639: Improvement -  Inherit from theme secondary button option support for Astra child themes.

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

[Buttons] Inherit from theme secondary button option for Astra child themes
https://brainstormforce.atlassian.net/browse/SPCTR-3639

### Screenshots

https://bsf.d.pr/v/zz17lf

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,6 +216,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 ## Changelog ##
 
 ### x.x.x ###
+* Improvement: Buttons - Added Inherit from theme secondary button option for Astra child themes.
```
Comment (bsf-zanev) [inline README.md:219]:
```suggestion
* Improvement: Buttons - Handled the Inherit-from-theme secondary-button option for Astra child themes.
```

File: classes/class-uagb-init-blocks.php [hunk 2]
```diff
@@ -798,6 +798,7 @@ function( $a, $b ) {
 				'uag_enable_gbs_extension'                => \UAGB_Admin_Helper::get_admin_settings_option( 'uag_enable_gbs_extension', 'enabled' ),
 				'current_theme'                           => wp_get_theme()->get( 'Name' ),
 				'is_gutenberg_activated'                  => is_plugin_active( 'gutenberg/gutenberg.php' ), // TODO: Once Gutenberg merged the rename functionality code in WP then we need to remove localization part for is_gutenberg_activated.
+				'astra_themes'                            => defined( 'ASTRA_THEME_SETTINGS' ),
```
Comment (bsf-zanev) [inline classes/class-uagb-init-blocks.php:801]:
```suggestion
				'is_astra_based_theme' => defined( 'ASTRA_THEME_SETTINGS' ),
```

File: src/blocks/buttons-child/settings.js [hunk 3]
```diff
@@ -148,6 +148,7 @@ const Settings = ( props ) => {
 	} = attributes;
 
 	const currentTheme = uagb_blocks_info.current_theme;
+	const astraThemes = uagb_blocks_info.astra_themes;
```
Comment (bsf-zanev) [inline src/blocks/buttons-child/settings.js:151]:
```suggestion
	const isAstraBasedTheme = uagb_blocks_info. is_astra_based_theme;
```

**Review by bsf-zanev — COMMENTED** [review]
Minor improvements for better understanding.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Updated.

File: classes/class-uagb-init-blocks.php (hunk 2 above)
Comment (rajmohanreddy774) [reply classes/class-uagb-init-blocks.php]: Updated.

File: src/blocks/buttons-child/settings.js (hunk 3 above)
Comment (rajmohanreddy774) [reply src/blocks/buttons-child/settings.js]: Updated.

**Review by bsf-zanev — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**